projects/spectator/src/lib/base/dom-spectator.ts

```diff
@@ -175,6 +175,30 @@ export abstract class DomSpectator<I> extends BaseSpectator {
     return event;
   }
 
+  public triggerEventHandler<C = any, K extends keyof C = any>(
```

Comment:
Nice to have: `K extends keyof C` - I think that we can create a type that'll extract only the keys that are of type `EventEmitter`.

Reply:
I had to upgrade my TS knowledge but it's done 👍